2017/12/15 12:26:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/PositionController.php(190): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): PositionController->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position/...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position/list
---
2017/12/15 12:26:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position2.php(108): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position2->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/Position2Controller.php(157): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): Position2Controller->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position2...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position2/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position2/list
---
2017/12/15 12:28:16 [error] [php] move_uploaded_file(/var/www/html/tms/protected/../upload/producer/78e98811a258121907603610153c19f7): failed to open stream: Permission denied (/var/www/html/tms/protected/controllers/admin/ProducerController.php:399)
Stack trace:
#0 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/html/framework/web/CController.php(286): ProducerController->runAction()
#2 /var/www/html/framework/web/CController.php(265): ProducerController->runActionWithFilters()
#3 /var/www/html/framework/web/CWebApplication.php(282): ProducerController->run()
#4 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/admin/producer/uploadPhotoDo
in /var/www/html/tms/protected/controllers/admin/ProducerController.php (399)
in /var/www/html/tms/index.php (31)
2017/12/15 12:28:41 [error] [php] move_uploaded_file(/var/www/html/tms/protected/../upload/producer/755cdfc75cf119a3dcfc808624dc58c5): failed to open stream: Permission denied (/var/www/html/tms/protected/controllers/admin/ProducerController.php:399)
Stack trace:
#0 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/html/framework/web/CController.php(286): ProducerController->runAction()
#2 /var/www/html/framework/web/CController.php(265): ProducerController->runActionWithFilters()
#3 /var/www/html/framework/web/CWebApplication.php(282): ProducerController->run()
#4 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/admin/producer/uploadPhotoDo
in /var/www/html/tms/protected/controllers/admin/ProducerController.php (399)
in /var/www/html/tms/index.php (31)
2017/12/15 19:04:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "adminLogin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('adminLogin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/adminLogin
---
2017/12/15 19:04:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2017/12/20 13:56:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1595): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/StatisticsController.php(61): CActiveRecord->count(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): StatisticsController->actionList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('list')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/statistic...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/statistics/list
HTTP_REFERER=http://192.168.1.169/tms/admin/monitor/index
---
2017/12/20 13:56:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position2.php(108): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position2->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/Position2Controller.php(157): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): Position2Controller->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position2...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position2/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position2/list
---
2017/12/20 13:56:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/PositionController.php(190): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): PositionController->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position/...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position/list
---
2017/12/20 13:56:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/PositionController.php(20): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): PositionController->actionOutside()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('outside')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position/...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position/outside
HTTP_REFERER=http://192.168.1.169/tms/admin/position/list
---
2017/12/20 13:56:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/PositionController.php(190): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): PositionController->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position/...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position/list
---
2017/12/20 13:56:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/PositionFloorController.php(259): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): PositionFloorController->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/positionF...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/positionFloor/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/positionFloor/list
---
2017/12/21 19:35:24 [error] [php] Undefined index: username (/var/www/html/tms/protected/controllers/SiteController.php:136)
Stack trace:
#0 /var/www/html/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#5 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/site/adminLoginDo
in /var/www/html/tms/protected/controllers/SiteController.php (136)
in /var/www/html/tms/index.php (31)
2017/12/25 12:15:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
HTTP_REFERER=http://192.168.1.169/tms/
---
2017/12/25 12:21:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "php_info".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('php_info')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/php_info
---
2017/12/25 13:07:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "php_info".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('php_info')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/php_info
---
2018/01/04 21:13:06 [error] [php] Undefined index: username (/var/www/html/tms/protected/controllers/SiteController.php:136)
Stack trace:
#0 /var/www/html/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#5 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/site/adminLoginDo
in /var/www/html/tms/protected/controllers/SiteController.php (136)
in /var/www/html/tms/index.php (31)
2018/01/08 11:24:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position.php(127): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1595): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/StatisticsController.php(61): CActiveRecord->count(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): StatisticsController->actionList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('list')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/statistic...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/statistics/list
HTTP_REFERER=http://192.168.1.169/tms/admin/admin/list
---
2018/01/08 11:24:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'sto'' in /var/www/html/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/html/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/html/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/html/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/html/framework/base/CModule.php(103): CModule->getComponent('dbSto')
#4 /var/www/html/tms/protected/models/Position2.php(108): CModule->__get('dbSto')
#5 /var/www/html/framework/db/ar/CActiveRecord.php(667): Position2->getDbConnection()
#6 /var/www/html/framework/db/ar/CActiveRecord.php(1469): CActiveRecord->getCommandBuilder()
#7 /var/www/html/tms/protected/controllers/admin/Position2Controller.php(157): CActiveRecord->findAll(Object(Criteria))
#8 /var/www/html/framework/web/actions/CInlineAction.php(49): Position2Controller->actionGetList()
#9 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/framework/web/CWebApplication.php(282): CController->run('getList')
#13 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/position2...')
#14 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/html/tms/index.php(31): CApplication->run()
#16 {main}
REQUEST_URI=/tms/admin/position2/getList
HTTP_REFERER=http://192.168.1.169/tms/admin/position2/list
---
2018/01/08 13:14:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/08 13:14:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/08 13:14:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
HTTP_REFERER=http://192.168.1.169/tms/
---
2018/01/12 13:22:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/12 13:22:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:23:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpMyAmin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('phpMyAmin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/phpMyAmin
---
2018/01/12 13:39:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpMyAmin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('phpMyAmin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/phpMyAmin
---
2018/01/12 13:39:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpMyAmin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('phpMyAmin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/phpMyAmin
---
2018/01/12 13:39:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:42:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpMyAmin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('phpMyAmin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/phpMyAmin
---
2018/01/12 13:50:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/12 13:50:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/12 13:50:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:51:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 13:51:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "phpMyAmin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('phpMyAmin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/phpMyAmin
---
2018/01/12 14:06:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/12 14:13:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 14:21:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin
---
2018/01/12 14:23:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('Admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/Admin
---
2018/01/12 14:23:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 14:24:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('admin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/admin/
---
2018/01/12 19:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "adminLogin".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('adminLogin')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/adminLogin
---
2018/01/12 21:02:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
HTTP_REFERER=http://192.168.1.109/tms/
---
2018/01/12 21:18:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/12 21:18:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/html/tms/index.php(31): CApplication->run()
#3 {main}
REQUEST_URI=/tms/favicon.ico
---
2018/01/15 18:06:56 [error] [php] copy(/var/www/html/tms/protected/../resource//map/3F.svg): failed to open stream: No such file or directory (/var/www/html/tms/protected/controllers/admin/FloorController.php:607)
Stack trace:
#0 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/html/framework/web/CController.php(286): FloorController->runAction()
#2 /var/www/html/framework/web/CController.php(265): FloorController->runActionWithFilters()
#3 /var/www/html/framework/web/CWebApplication.php(282): FloorController->run()
#4 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/admin/floor/updateDo
in /var/www/html/tms/protected/controllers/admin/FloorController.php (607)
in /var/www/html/tms/index.php (31)
2018/01/15 18:53:04 [error] [php] fopen(/var/www/html/tms/protected/../resource//map/3F.svg): failed to open stream: No such file or directory (/var/www/html/tms/protected/controllers/admin/FloorController.php:178)
Stack trace:
#0 /var/www/html/framework/web/CController.php(308): CInlineAction->runWithParams()
#1 /var/www/html/framework/web/CController.php(286): FloorController->runAction()
#2 /var/www/html/framework/web/CController.php(265): FloorController->runActionWithFilters()
#3 /var/www/html/framework/web/CWebApplication.php(282): FloorController->run()
#4 /var/www/html/framework/web/CWebApplication.php(141): CWebApplication->runController()
#5 /var/www/html/framework/base/CApplication.php(180): CWebApplication->processRequest()
#6 /var/www/html/tms/index.php(31): CWebApplication->run()
REQUEST_URI=/tms/admin/floor/viewer?id=3
in /var/www/html/tms/protected/controllers/admin/FloorController.php (178)
in /var/www/html/tms/index.php (31)
